tests/baselines/reference/numLit.errors.txt

```diff
@@ -1,13 +0,0 @@
-tests/cases/compiler/numLit.ts(3,3): error TS1005: ';' expected.
-tests/cases/compiler/numLit.ts(3,3): error TS2304: Cannot find name 'toString'.
-
-
-==== tests/cases/compiler/numLit.ts (2 errors) ====
-    1..toString();
-    1.0.toString();
-    1.toString();
-      ~~~~~~~~
-!!! error TS1005: ';' expected.

```

Comment:
It technically could be allowed, since the emit is ```1 .toString();```

Firefox says 'SyntaxError: identifier starts immediately after numeric literal', could add any error message really...